readme.md

```diff
@@ -0,0 +1,52 @@
+# GMod-Seamless-Portals
+[![](https://img.youtube.com/vi/lgiPHZdTGxs/0.jpg)](http://www.youtube.com/watch?v=lgiPHZdTGxs "")
+
+### Description
+My seamless portals addon for garrys mod. This is a mod that adds a entity called the Seamless Portal, and a functioning portal gun!
+You can spawn the Portals in the spawnmenu, or by using the portal gun in the weapons tab.
+
+### Installation
+To install this addon, download and unzip the contents and put it in your garrys mod addon folder
+
+### Features
+ * More seamless
+ * Working wall & floor portals
+ * Prediction for multiplayer & players with high ping
+ * Floor extrusion if your player gets stuck in the ground
+ * Sounds travel through
+ * Black skybox fix
+ * Optimized traceline implementation
+ * Includes non janky portal gun
+ * Includes some tools to play with
```

Comment:
Like? This is the kind of stuff you put on features

Reply:
I did not write the original text